.github/workflows/image-publish.yaml

```diff
@@ -22,6 +22,12 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Duplicate of https://github.com/nirmata/kyverno-notation-aws/pull/142/files#diff-e8d2c07eb29c755b9112bc088146f8fac5bbc09adee367a0a25adb9ae303a5edR35-R40.

Reply:
removing it